entrypoint/clusterreset/clusterreset_test.go

```diff
@@ -0,0 +1,93 @@
+package clusterreset
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("Test:", func() {
+
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Nodes", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus(),
+		)
+	})
+
+	It("Verifies ClusterIP Service", func() {
+		testcase.TestServiceClusterIp(true, true)
```

Comment:
why here params ( true,true) ?

Reply:
The ClusterIP workload is being deleted before the cluster reset to show that it is not still present after the cluster reset has been performed. Whereas with Nodeport that is to test that the workload is still present after the cluster reset.